test/e2e/autorepair_test.go

```diff
@@ -33,7 +33,7 @@ func TestAutoRepair(t *testing.T) {
 	clusterOpts.NodePoolReplicas = 3
 	clusterOpts.AutoRepair = true
 
-	hostedCluster := e2eutil.CreateAWSCluster(t, ctx, client, clusterOpts, globalOpts.ArtifactDir)
+	hostedCluster := e2eutil.CreateCluster(t, ctx, client, &clusterOpts, hyperv1.AWSPlatform, globalOpts.ArtifactDir)
```

Comment:
should hyperv1.AWSPlatform be set dynamically?

Reply:
Maybe, but I'm trying not to give _too_ much consideration to comprehensive cross-platform consideration yet. We are stretched so thin just trying to go live with AWS... mostly my goal is to get what we have maintainable for this team, with AWS primarily. We have to examine the whole CLI and e2e scaffold top to bottom to really get it right, IMO. In the meantime it's an exercise in controlling organic growth and premature abstraction...